projects/igniteui-angular/src/lib/grids/filtering/excel-style/grid.excel-style-filtering.component.html

```diff
@@ -25,6 +48,7 @@ <h4 class="igx-typography__h6">{{ column.header || column.field }}</h4>
         <ng-template #defaultExcelStyleMovingTemplate>
             <igx-excel-style-column-moving
                 class="igx-excel-filter__move"
```

Comment:
Shouldn't you remove this class as it will be conditionally applied by `[ngClass]`?